test/Libraries/TestServices/TestPathResolver.cs

```diff
@@ -11,12 +11,16 @@ public class TestPathResolver : IPathResolver
         private readonly List<string> additionalResolutionPaths;
         private readonly List<string> additionalNodeDirectories;
         private readonly List<string> preloadedLibraryPaths;
+        private readonly string userDataRootFolder;
+        private readonly string commonDataRootFolder;
 
         public TestPathResolver()
         {
             additionalResolutionPaths = new List<string>();
             additionalNodeDirectories = new List<string>();
             preloadedLibraryPaths = new List<string>();
+            userDataRootFolder = null;
+            commonDataRootFolder = null;
```

Comment:
These members are not needed, simply return `string.Empty` from the `get` would do.